enos/modules/solo_worker/templates/worker.hcl

```diff
@@ -0,0 +1,28 @@
+listener "tcp" {
+  purpose = "proxy"
+  tls_disable = true
+  address = "0.0.0.0"
+}
+
+worker {
+  # Name attr must be unique across workers
+  name = "worker-${id}"
+  description = "Enos Boundary worker ${id}"
+
+  # Workers must be able to reach controllers on :9201
+  controllers = ${controller_ips}
+
+  public_addr = "${public_addr}"
+
+  tags {
+    region = "["${region}"]"
+    type = ${type}
+  }
+}
+
+# must be same key as used on controller config
+kms "awskms" {
+  purpose    = "worker-auth"
+  region     = "${region}"
```

Comment:
nit: does this need quotes still?

Reply:
I believe so, yes.

String templates don't interpolate strings with their quote sigils https://developer.hashicorp.com/terraform/language/expressions/strings#string-templates

And the HCL blocks don't use quotes for the keys, but they do for the values https://developer.hashicorp.com/boundary/docs/configuration/worker/kms-worker

Does that read the same to you? Or maybe there's something else that I'm missing.